runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -193,7 +188,7 @@ Java_java_lang_StackWalker_getImpl(JNIEnv *env, jobject clazz, jlong walkStateP)
 	if (J9_ARE_NO_BITS_SET((UDATA) (walkState->userData1), FRAME_VALID)) {
 		/* skip over the current frame */
 		walkState->userData1 = (void *) ((UDATA)walkState->userData1 & FRAME_FILTER_MASK);
-		if (J9_STACKWALK_RC_NONE != vm->walkStackFrames(vmThread, walkState)) {
+		if (J9_STACKWALK_RC_NONE != vm->internalVMFunctions->commonWalker(vmThread, walkState)) {
```

Comment:
please add a comment here to explain the use of `commonWalker` (and relation to `J9_STACKWALK_RESUME` flag)

Reply:
It was a mistake to use `commonWalker` here.